src/configs/lang.ts

```diff
@@ -411,3 +411,20 @@ export const IRS_GREY_THRESHOLD = translate('IRS_GREY_THRESHOLD', 'Grey');
 export const IRS_RED_THRESHOLD = translate('RED_THRESHOLD', 'Red');
 export const IRS_YELLOW_THRESHOLD = translate('YELLOW_THRESHOLD', 'Yellow');
 export const IRS_LIGHT_GREEN_THRESHOLD = translate('LIGHT_GREEN_THRESHOLD', 'Light Green');
+export const FOCI_OF_RESIDENCE = translate('FOCI_OF_RESIDENCE', 'Foci of Residence');
+export const FOCI_OF_INFECTION = translate('FOCI_OF_INFECTION', 'Foci of Infection');
+export const CASE_DETAILS = translate('CASE_DETAILS', 'Case Details');
+export const CASE_INFORMATION = translate('CASE_INFORMATION', 'Case Information');
+export const AGE = translate('AGE', 'Age');
+export const SURNAME = translate('SURNAME', 'Surname');
+export const PATIENT_NAME = translate('PATIENT_NAME', 'Patient Name');
+export const HOUSE_NUMBER = translate('HOUSE_NUMBER', 'House Number');
+export const INVESTIGATION_DATE = translate('INVESTIGATION_DATE', 'Investigation Date');
+export const SPECIES = translate('SPECIES', 'Species');
+export const DIAGNOSIS_DATE = translate('DIAGNOSIS_DATE', 'Date of Diagnosis');
+export const ID = translate('ID', 'Id');
```

Comment:
I thought we already had one for ID (which, in TH was "translated" to ID). Is it important that the `d` is lowercase? Should it be uppercase b/c it's an abbreviation and not [Freud's Id](https://en.wikipedia.org/wiki/Id,_ego_and_super-ego#Id)?

Reply:
@cKellyDesign It actually needs to be ID, but i could not find it in `config/lang.ts`

Reply:
Scratch that, found it.